pkgs/applications/networking/browsers/telescope/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, autoreconfHook
+, bison
+, libevent
+, libressl
+, ncurses
+}:
+
+
+stdenv.mkDerivation rec {
+  pname = "telescope";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "omar-polo";
+    repo = "telescope";
+    rev = "87e3e8018ef2858201ca3641eff06e130ea3ccd9";
```

It looks like you're not fetching the release specified in the `version` attribute, but instead an unstable commit.
```suggestion
    rev = version;
```
When using this, `fetchFromGitHub` will fetch the version specified.